x/participation/keeper/msg_participate_test.go

```diff
@@ -54,30 +60,41 @@ func Test_msgServer_Participate(t *testing.T) {
 		{
 			name: "valid message tier 1",
 			msg: &types.MsgParticipate{
-				Participant: addrWithDelsTier1,
-				AuctionID:   auctionID,
+				Participant: addrsWithDelsTier[0],
+				AuctionID:   auctionID1,
 				TierID:      1,
 			},
 			desiredUsedAlloc:      1,
-			currentAvailableAlloc: availableAllocTier1,
+			currentAvailableAlloc: availableAllocsTier[0],
 			blockTime:             validRegistrationTime,
 		},
 		{
 			name: "valid message tier 2",
 			msg: &types.MsgParticipate{
-				Participant: addrWithDelsTier2,
-				AuctionID:   auctionID,
+				Participant: addrsWithDelsTier[1],
+				AuctionID:   auctionID1,
 				TierID:      2,
 			},
 			desiredUsedAlloc:      2,
-			currentAvailableAlloc: availableAllocTier2,
+			currentAvailableAlloc: availableAllocsTier[1],
 			blockTime:             validRegistrationTime,
 		},
+		{
+			name: "should allow participation when blockTime < registrationPeriod",
```

Comment:
Mmh not sure about the name of the test case, in this case we try to participate when [unix(0) - startTime] is shorter than registration period?

If this is the case, I would suggest `should allows participating when registration period is longer than range between Unix time 0 and auction's start time`

Which is a bit long but I thing it is fine for test maintainability